### PR TITLE
Consolidation of `services/health/*` and `service/service.go`

### PR DESCRIPTION
**Pull request by pinebit**
Closes https://app.shortcut.com/chainlinklabs/story/23383/consolidate-service-and-health-packages

### CODE REVIEW

File: core/chains/evm/chain.go
Comment (archseer) [inline core/chains/evm/chain.go:17]: The split between `services`/`service` was there because `services` will pull in a few more services (only balance monitor, prom reporter, it's a lot better now that the v1 code was removed). They should be moved away from the root crate then.

File: core/cmd/client.go
Comment (archseer) [inline core/cmd/client.go:198]: Why is this under `services` now?

Comment (pinebit) [reply core/cmd/client.go:198]: Because of a circular dependency: services -> pg -> services

File: core/chains/evm/chain.go
Comment (pinebit) [reply core/chains/evm/chain.go:17]: This I did not understand. Can you please clarify a little bit more what should I do? Thanks!

Comment (pinebit) [reply core/chains/evm/chain.go:17]: Do you mean `balance_monitor.*` and `prom_reporter.*` should be moved away from ./services root to respective subfolders?

File: core/cmd/client.go
Comment (archseer) [reply core/cmd/client.go:198]: Yeah so this is what I mentioned on the IPM, I ended up having a separate package because of this

File: core/chains/evm/chain.go
Comment (archseer) [reply core/chains/evm/chain.go:17]: Yeah, nothing should be in the root package other than the service types.

Comment (pinebit) [reply core/chains/evm/chain.go:17]: Done.

File: core/cmd/client.go
Comment (pinebit) [reply core/cmd/client.go:198]: Right, and in order to fix this, I had to move EventBroadcaster out from pg. Is there a better idea to do this? Or do you believe EventBroadcaster should belong to pg?

Comment (archseer) [reply core/cmd/client.go:198]: If you remove all services from the root `services` crate, then nothing will import `pg` there and the circular dependency will be gone

Comment (pinebit) [reply core/cmd/client.go]: All done

**Review by samsondav — APPROVED** [review]

**Comment by pinebit** [comment]
@archseer this PR will be waiting for your final approval before I merge it.

**Pull request merged**
